### ISSUE TITLE
A4Tech Venus2.0 not recognized on Ubuntu 14.4 Trusty Tahr 32-bit

### ISSUE THREAD

**Issue by liakos** [wireapp/wire-desktop#1108]
I can't choose my external webcam as an option in the audio/video preferences. It doesn't show up as an option in the camera tab.

Everything works fine on my Windows 7 installation of  Wire.

**Comment by mythsunwind** [comment]
Please make sure that this is not a general issue with Linux support of your webcam. Please install Cheese with `sudo apt-get install cheese` and check if your webcam works in Cheese.

**Comment by liakos** [comment]
I think the web cam is not listed in the linux uvc project...

Here is the result of lsusb:

Bus 001 Device 002: ID 0ac8:3420 Z-Star Microelectronics Corp. Venus USB2.0 Camera

Nonetheless, the camera works fine on Cheese.
.......

However, something odd happened..
As soon as I checked the webcam on Cheese, I decided to open Wire again and see if the webcam was recognized this time.
And for some reason that I don't understand, the webcam WAS recognized.
I decided to shut down and boot again some time later to see if it was still recognized and it was!

So, I don't know why, but as soon as I checked the webcam with Cheese, it was recognized in Wire, too.
I don't understand this but, the point is, that my issue is solved now.
If you have any idea how this could happen, I would be very much interested to know how.

Thank you for your help!

**Comment by liakos** [comment]
I found something interesting.
I tried plugging out the webcam and I could still see the webcam as an option on the "Camera" tab on "Audio/Video Preferences". Of course there was a black screen when I chose it.

Oddly enough, when I plugged it back in, the black screen was still there.
I opened Cheese and I saw the same black screen.
So, I restarted the system and Cheese shows the message "There was an error playing video from the webcam".

I guess that's a Cheese issue now so, it's irrelevant to this post and I also don't use Cheese so I don't care.

The good thing is that after the reboot the camera works fine again on Wire.
So, to summarize, I think there is a problem (or bug) with Wire when you plug out and back in the webcam. You have to reboot in order for Wire to recognize the webcam again.

Just thought I should let you know.
